### DISCUSSION TITLE
DSP announced for Game Pass.  Will this be compatible?

### DISCUSSION THREAD

**Discussion by thw0rted** [NebulaModTeam/nebula#584] [Q&A] [answered]
I've had my eye on DSP for a while, but never bought it on Steam -- nothing personal, I just usually avoid Early Access titles.  I saw that it's coming to Game Pass next month.  I'm mostly interested in playing the game with my son.  Will this mod work on the Game Pass PC / MS Store version?

**Comment by PhantomGamers** [comment, marked as answer]
If it supports the Xbox app's advanced installation and management features then Nebula should work with it, yes.

**Reply by thw0rted** [reply]
Just what I was looking for, thanks!  I'll definitely try it out after launch.
